src/npc_arena1v1.h

```diff
@@ -60,7 +60,7 @@ static bool Arena1v1CheckTalents(Player* player)
 
     if (count >= 36)
     {
-        ChatHandler(player->GetSession()).SendSysMessage("Du kannst nicht beitreten, da du zuviele Talentpunkte in einem verbotenen Baum hast. (Heal / Tank)");
+        ChatHandler(player->GetSession()).SendSysMessage("You can not join because you have too many talent points in a forbidden tree. (Heal / Tank)");
```

Comment:
Too many talent points in a forbidden tree ? if it's forbidden, it's 0 talent allowed in that tree no ? if so then the phrase is redundant :P

Reply:
To tell you truth , I’m not sure why this is set like this either , or how I would set it , I just converted the German to English on this line , appreciate and input others have here in this.

Reply:
ok on further research this is a conf setting regarding extra points "Description: If true, healers can't join 1v1 arena, if they invested more than 35 talent points in a healing-talent tree." 

so this is possible adjusted +1 to compensate for something or this was a custom change to add 36 not 35 as originally intended